crates/astria-sequencer/src/accounts/query.rs

```diff
@@ -1,49 +1,37 @@
 use std::collections::VecDeque;
 
-use anyhow::{
-    anyhow,
-    bail,
-    Context as _,
-    Result,
-};
+use anyhow::{anyhow, bail, Context as _, Result};
 use borsh::BorshSerialize as _;
 use tracing::instrument;
 
-use crate::accounts::{
-    state_ext::StateReadExt,
-    types::{
-        Address,
-        Balance,
-        Nonce,
-    },
-};
+use crate::accounts::state_ext::StateReadExt;
 
 #[derive(Debug)]
-pub enum QueryRequest {
-    BalanceQuery(Address),
-    NonceQuery(Address),
+pub(crate) enum QueryRequest {
+    BalanceQuery(String),
+    NonceQuery(String),
```

Comment:
removal of the `Address` and `Balance` types are going to be annoying because I have to re-add them in my next PR for accounts that are key-backed, not just arbitrary strings. `Balance` also has to be a custom type because protobuf doesn't support u128 :/ can you add these types back? alternatively I'll have to re-add them in my next PR

Reply:
brought them back. I also added a `Uint128` protobuf type in #77.